app/controllers/leads_controller.rb

```diff
@@ -121,12 +121,13 @@ def text
   def autotext
         @lead = Lead.find(params[:id])
         first_name = @lead.first_name.split(' ')[0]
+        auto_text = has_auto_text?
```

Comment:
Just a heads up, when a method ends in a ? - that generally means it is going to return true or false. Maybe think about calling the method something else such as `admin_autotext`